### PR TITLE
Add normalize as an argument

### PR DESCRIPTION
**Pull request by Eleven1Liu**
**Description**  Add `--normalize` to config to decide if we should normalize the word embeddings by:
```python
vector = vector  / float(np.linalg.norm(vector) + 1e-6)
```

### CODE REVIEW

File: libmultilabel/data_utils.py [hunk 1]
```diff
@@ -121,14 +121,32 @@ def load_or_build_text_dict(
     min_vocab_freq=1,
     embed_file=None,
     embed_cache_dir=None,
-    silent=False
+    silent=False,
+    normalize=True
```
Comment (Eleven1Liu) [inline libmultilabel/data_utils.py:125]: set `False`

File: main.py [hunk 2]
```diff
@@ -70,6 +70,8 @@ def get_config():
                         help='Momentum factor for SGD only (default: %(default)s)')
     parser.add_argument('--patience', type=int, default=5,
                         help='Number of epochs to wait for improvement before early stopping (default: %(default)s)')
+    parser.add_argument('--normalize', type=bool, default=False,
```
Comment (sian-chen) [inline main.py:73]: Use `action='store_true'` for flags.
https://docs.python.org/3/library/argparse.html#action

```suggestion
    parser.add_argument('--normalize', action='store_true',
```

**Review by sian-chen — CHANGES_REQUESTED** [review]

**Review by sian-chen — APPROVED** [review]

File: main.py [hunk 3]
```diff
@@ -70,6 +70,8 @@ def get_config():
                         help='Momentum factor for SGD only (default: %(default)s)')
     parser.add_argument('--patience', type=int, default=5,
                         help='Number of epochs to wait for improvement before early stopping (default: %(default)s)')
+    parser.add_argument('--normalize', action='store_true',
+                        help='Whether the word embeddings divide by `float(np.linalg.norm(vector) + 1e-6)` (default: %(default)s)')
```
Comment (cjlin1) [inline main.py:74]: this description isn't good. "vector" is unclear here and it's not good to have the code np.xxx in the help.
I think we should just say
Whether the embeddings of each word is normalized to a unit vector

File: libmultilabel/data_utils.py [hunk 4]
```diff
@@ -140,7 +158,7 @@ def load_or_build_text_dict(
 
     if os.path.exists(embed_file):
         logging.info(f'Load pretrained embedding from file: {embed_file}.')
-        embedding_weights = get_embedding_weights_from_file(vocabs, embed_file, silent)
+        embedding_weights = get_embedding_weights_from_file(vocabs, embed_file, silent, normalize)
```
Comment (cjlin1) [inline libmultilabel/data_utils.py:161]: so the "normalize" option is only for user-specified embeddings?  If I am a user I would think that this is applied to all situations. Need some discussion tomorrow

**Pull request merged**